modules/docs/mdx/6.0-MIGRATION-GUIDE.mdx

```diff
@@ -364,3 +377,71 @@ will have two effects for this component:
   - This was previously happening only on viewports < 576px wide
 
 These changes in behavior are intentional and expected.
+
+## Button Updates
+
+`PrimaryButton`, `SecondaryButton`, and `TertiaryButton` have been slightly adjusted to tighten up
+the spacing and use our new type hierarchy. We also adjusted the opacity for their disabled state
+for a bit more consistency. These changes are automatic when you upgrade and do not require any
+manual updates in your codebase. However, these changes will likely cause any automated visual
+regression tests to fail, and you will need to update them accordingly. These adjustments are fairly
+minor, and we do not expect them to create significant layout shifts in your UI. But, as with any
+visual update, you will want to review your UI to ensure there are no adverse effects. We also added
+some net-new styling: `PrimaryButton`, `SecondaryButton`, and `TertiaryButton` have a new
+extra-small size, and `PrimaryButton` has a new `inverse` variant.
+
+All these changes are described in detail in the sections below.
+
+### Primary Button
+
+#### Primary Disabled State
+
+Previously the button's disabled state had a `blueberry200` background, but it now uses the default
+`blueberry400` and sets the entire button to 40% opacity. This creates more more consistency for the
+disabled states across all our buttons.
+
+#### Primary Large
+
+The padding for large `PrimaryButton`s has been slightly adjusted for more visual balance and to
+take up a bit less layout space. Specifically we:
+
+- decreased the space between the button icon and text from `12px` to `8px`
+- decreased the space between the button container and the icon from `28px` to `24px`
+
+#### Primary Medium
+
+The icon size for medium `PrimaryButton`s has been decreased from `24px` to `20px`, but the overall
+height of the button will remain the same.
+
+#### Primary Small
+
+We now support icons for small `PrimaryButton`s. The icons will be `20px x 20xpx`.
+
+#### Primary Extra Small
+
+We added a new size, `extraSmall` to our `PrimaryButton`s. These are particularly helpful for use
+cases where you have dense UI or tight layout constraints such as tables.
+
+### Secondary Button
+
+#### Secondary Large
+
+The padding for large `SecondaryButton`s has been slightly adjusted for more visual balance and to
```

Comment:
Do you think it's worth it to call out all of these identical changes for each type of button? Just wondering if brevity or clarity adds more value here.

Reply:
I put an overview section at the top under "Button Updates" in the Primary Button PR. I would like to keep these sections specific, so if a developer or QA needs to use it as a reference for their visual regression testing, they can easily see the specific changes per button + size.